Changes

```diff
@@ -1,5 +1,10 @@
 Revision history for MojoX-Renderer-Xslate
 
+{{NEXT}}
```

Comment:
Is this line required?

Reply:
The {{$NEXT}} market is used by Dist::Zilla when building the release artefact to highlight the changes since the last release. We normally leave the marker, followed by a linebreak, to show that there are no pending changes since the latest version.